test/cql-pytest/test_tablets.py

```diff
@@ -52,3 +52,15 @@ def test_alter_cannot_change_vnodes_to_tablets(cql, skip_without_tablets):
     with new_test_keyspace(cql, ksdef) as keyspace:
         with pytest.raises(InvalidRequest):
             cql.execute(f"ALTER KEYSPACE {keyspace} WITH replication = {{'class': 'NetworkTopologyStrategy', 'replication_factor': 1}} AND tablets = {{'initial': 1}};")
+
+
+def test_alter_doesnt_enable_tablets(cql, skip_without_tablets):
```

Comment:
Again, this test mostly enshrines the current behavior - I wonder if we really want to keep it this way in the future. If you think what I'm saying is true, perhaps a comment should say that.

Reply:
Currently this is how things work. If we're going to allow vnode->tablets conversion some day, this test will need to be fixed of course

Reply:
Yes, I know this is how things work, but now is the time to at least talk about whether we **want** it to work this way or it's just a temporary fact of life. And if it's the latter, I think the test should have a comment saying this. I hate it when some dtest fails and it turns out they enshrined some wrong behavior but now I need to figure this out - if it was really wrong behavior or really intended.